ufl/finiteelement/mixedelement.py

```diff
@@ -292,8 +295,8 @@ def __init__(self, family, cell=None, degree=None, dim=None,
         # Initialize element data
         MixedElement.__init__(self, sub_elements, value_shape=value_shape,
                               reference_value_shape=reference_value_shape)
-        # FIXME: Storing this here is strange, isn't that handled by
-        # subclass?
+        FiniteElementBase.__init__(self, sub_element.family(), cell, degree, quad_scheme,
+                                   value_shape, reference_value_shape)
         self._family = sub_element.family()
         self._degree = sub_element.degree()
         self._sub_element = sub_element
```

Comment:
Check this.

Reply:
I've changed this such that `FiniteElementBase.__init__` sets everything except `self._sub_element`. (see rebased commit 93a9194e9a2fdac6b5c4101f4c4005a9fcbaa7db ) Note that it now sets `self.degree` to `sub_element.degree()` in `FiniteElementBase.__init__`.